pyproject.toml

```diff
@@ -12,67 +12,67 @@ write-schema-json = "visivo.generate_project_schema_json:write_schema_json"
 
 [tool.poetry.dependencies]
 python = " >=3.10,<3.13"
-pydantic = ">=2.9.0,<=2.9.2"
```

Comment:
I believe we had an issue with >2.9.2

Reply:
Understood. Can keep it this way for now but looks like a better long-term solution would be to resolve that issue in the first place

Reply:
@jpjsonic Do you remember what the issue was?

Reply:
I went and searched slack and didn't find anything.